build.gradle

```diff
@@ -33,3 +33,23 @@ task generateKeys {
     }
   }
 }
+
+task command (type: JavaExec) {
+  group = 'CAS'
+  description = "Invokes the CAS Command Line Shell. Use -q to quiet Gradle output. To pass arguments, use '-Pcargs=arg1,arg2,arg3,...'"
```

Comment:
Unfortunately naming it "args" (Pargs) does cause a conflict with the same variable used in build.gradle as well as the args value for this Java task. Additionally, arguments for the command shell can't be passed from the initial scripts to gradle since they conflict with existing gradle commands.